man/systemd.exec.xml

```diff
@@ -1113,8 +1113,8 @@
         also <filename>/usr/lib/modules</filename> is made inaccessible. For this
         setting the same restrictions regarding mount propagation and privileges
         apply as for <varname>ReadOnlyPaths=</varname> and related calls, see above.
-        Note that limited automatic module loading due to user configuration or kernel
-        mapping tables might still happen as side effect of requested user operations,
+        Note that limited automatic module loading due to user or network configurations or
+        kernel mapping tables might still happen as side effect of requested user operations,
```

Comment:
well, "user configuration" in this regard was meant to mean "any kind of configuration applied by the user", which includes "network configuration"... I am not sure this change is really an improvment.

Also, "configurations" in plural is strange, and having a concatenation of three phrases via two "or"s is weird, no?